### ISSUE TITLE
Safe shutdown not working but safe reset is working Recalbox RPI 3 B+

### ISSUE THREAD

**Issue by ErnestoArtigas** [RetroFlag/retroflag-picase#45]
Hey everyone ! I've bought a NesPi+ from Kubii, a french seller, but I can't make the safe shutdown work... Safe reset is working, but not safe shutdown, it is as unpressed, I need to manually change the switch position to off to shutdown it...

**Comment by Fozziefett** [comment]
Also seeing the same problem with MegaPi on latest script. Reset works, shutdown does not. If I switch to off and reset, it wont restart and will remain off.

**Comment by LoonyToonz** [comment]
I have found if I reset then once it basicallly shuts down and before it starts I can use the power witch to witch off, but If I use power to witch off with the system running nothing happens.

**Comment by Fozziefett** [comment]
Yes, this is the same behavior a few of us experience with the unaltered
RetroFlag script. Glad to hear that you see the same thing.
